Releasing.md

```diff
@@ -196,11 +196,19 @@ For the body of the post, just copy this checklist and again replace all occurre
 
 <!-- wp:group -->
 <div class="wp-block-group"><!-- wp:paragraph -->
-<p>o Once everything is merged, send a message similar to the following to our friends in the <code>#platform9</code> Slack channel. If the release is a beta/hot fix (e.g. X.XX.2), be sure to directly mention the relevant Excellence Wranglers for the release and modify the following template as needed.</p>
-<!-- /wp:paragraph -->
-
+<p>o Once everything is merged, send a heads up to our friends in the <code>#platform9</code> Slack channel. If the release is for the frozen WPiOS and WPAndroid release branches (i.e. this isn't a beta/hot fix, e.g. X.XX.2), the message will look similar to the following:</p>
```

Comment:
> If the release is for the frozen WPiOS and WPAndroid release branches (i.e. this isn't a beta/hot fix, e.g. X.XX.2)

The regular releases are merged into `develop`, so maybe we should say here if the release IS NOT for the frozen WPiOS and WPAndroid release branches, wdyt?

Reply:
@fluiddot, this first part is intended to be about the regular release and is connected to the message that mentions merging into `develop`. So, I _think_ it's correct as it is. I've then split the message for beta fixes into a separate message below. Could you double-check to let me know if I'm missing something here? (I think the confusion here backs up the need for this list to eventually be automatically generated, based on whether the release is a beta or not.)

Reply:
> @fluiddot, this first part is intended to be about the regular release and is connected to the message that mentions merging into develop. So, I think it's correct as it is.

Ok, the part that confused me is that it states that the release (I understand the regular release) is for the frozen WPiOS and WPAndroid release branches. I'm not sure if we're referring to the branches `release/X.Y.Z` in WPiOS and WPAndroid repositories, if that's the case, then we should reference the `develop` branch as the WPiOS and WPAndroid release branches are not created at this point, right? Sorry because I might be overlooking something 😄 .

> I've then split the message for beta fixes into a separate message below. Could you double-check to let me know if I'm missing something here? (I think the confusion here backs up the need for this list to eventually be automatically generated, based on whether the release is a beta or not.)

I like the idea of splitting them to make it more clear and both messages look great to me 👍 . And yeah, once we generate the list automatically, we'll have a clearer and concise release checklist 😅 .

Reply:
Ah, you're right, sorry, I was getting mixed up with the differences 🤦‍♀️ I've updated now in https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/pull/75/commits/0c84520f764d342b256f4a7935fad328917de082! Does the updated wording make sense to you? (and thank you!)

Reply:
No worries 😊, now the message is perfect, thanks for updating it 🙇 .